packages/health-check/src/notifyChanges.js

```diff
@@ -0,0 +1,47 @@
+const db = require("./db");
+const { sendMessageToHealthCheckChannel, getRoleByName } = require("./discord");
+
+const { PORTAL_NAME = "PORTAL_NAME not defined" } = process.env;
+
+const notificationTimestamps = { critical: 0, verbose: 0 };
+const notificationUnchangedInterval = 24 * 60 * 60 * 1000; // 24 hours
+
+function notifyChanges(type) {
```

Comment:
The code is definitely readable but some comments would be nice. A small introduction on what it is you're trying to achieve.